### PR TITLE
Switched to ROW_FORMAT DYNAMIC for better compatibility with MariaDB 10.3 / 10.11

### PR DESCRIPTION
**Pull request by alexh-swdev**
### Description (*)

 I increased the version of `Mage/Catalog` to run a database migration script to alter the tables `catalog/product_relations` and `catalog_product_website` to not use `ROW_FORMAT=FIXED` anymore.

I decided to instead use `ROW_FORMAT=DYNAMIC` because it seems to be the standard on MariaDB: https://mariadb.com/kb/en/innodb-dynamic-row-format/

For more, see the discussion https://github.com/OpenMage/magento-lts/discussions/3962

### Fixed Issues (if relevant)

Database fails to get imported to MariaDB 10.3.* and 10.11.* if InnoDB tables have `ROW_FORMAT=FIXED`

### Questions or comments

I did manual tests on MariaDB 10.3 and 10.11. The results are looking good to me. (Meaning: the migration ran throug with success, the shops are still working :) , no apparent error messages or strange behavior)

I guess this should also work on MySQL but I don't have any at hand for testing. So maybe, before approving it, somenone wants to do some testing here.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All automated tests passed successfully (all builds are green)
 - [ ] Add yourself to contributors list

### CODE REVIEW

File: app/code/core/Mage/Catalog/sql/catalog_setup/upgrade-1.6.0.0.19.1.6-1.6.0.0.19.1.7.php
Comment (fballiano) [inline app/code/core/Mage/Catalog/sql/catalog_setup/upgrade-1.6.0.0.19.1.6-1.6.0.0.19.1.7.php:23]:
```suggestion
$installer->endSetup();

```

**Comment by fballiano** [comment]
Is this also supposed to fix new install? cause if I'm not mistaken it seems to fix only already installed projects

**Comment by alexh-swdev** [comment]
I only had already installed shops in my mind. I'll check against a new shop in the next couple of days.

**Comment by fballiano** [comment]
- tested the upgrade scripts on mysql 8.3, they work ok
- modified all sql scripts too for new installations (will test soon)

**Review by fballiano — APPROVED** [review]
approving but waiting for more feedback before merging

**Comment by fballiano** [comment]
since nobody pitched in, I'll merge.

**Pull request merged**
